available-connectors.json

```diff
@@ -10,7 +10,8 @@
     "settings": {
       "localStorage": {
         "type": "string",
-        "description": "window.localStorage key to use for persistence (browser only)"
+        "description":
+          "window.localStorage key to use for persistence (browser only)"
```

Comment:
Please revert all white-space-only changes. Did you use Prettier to format this file? It formats JSON files differently than `JSON.stringify` does :( But! The recently-released Prettier version 1.13 support json-stringify format, see https://prettier.io/blog/2018/05/27/1.13.0.html#added-json-stringify-parser-for-jsonstringify-style-formatting-4450-by-ikatyang.